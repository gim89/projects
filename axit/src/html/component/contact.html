<section id="contact" class="contact section__padding section__text">
  <div class="contact__container container">
    <h2 class="contact__title title__transform title--after">contact us</h2>
    <p class="contact__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
    <form action="#" class="contact__form">
      <div class="contact__input">
        <input type="text" name="name" class="contact__name" placeholder="Name" required>
        <input type="email" name="email" class="contact__email" placeholder="Email" required>
        <input type="text" name="subject" class="contact__subject" placeholder="Subject" required>
      </div>
      <textarea class="contact__message" name="message" placeholder="Message"></textarea>
      <input type="submit" class="contact__btn btn btn--orange btn--hover btn--active" value="Download">
    </form>
  </div>
</section>