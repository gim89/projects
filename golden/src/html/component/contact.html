<section class="contact section__padding">
  <div class="container">
    <h2 class="contact__title">about us</h2>
    <p class="subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
    <form action="#" class="contact__form">
      <div class="contact__item">
        <div class="contact__inputs">
          <input type="text" name="name" class="contact__name" placeholder="your name*" required>
          <input type="email" name="email" class="contact__email" placeholder="your e-mail*" required>
          <input type="text" name="subject" class="contact__subject" placeholder="subject">
        </div>
        <textarea name="message" class="contact__message" placeholder="your message"></textarea>
      </div>
      <div class="contact__item">
          <input type="submit" class="contact__btn btn btn--hover btn--active" value="Send Message">
      </div>
    </form>
  </div>
</section>